src/repr/src/row.rs

```diff
@@ -110,11 +110,14 @@ pub static VARIABLE_LENGTH_ENCODING: AtomicBool = AtomicBool::new(false);
 /// Rows are dynamically sized, but up to a fixed size their data is stored in-line.
 /// It is best to re-use a `Row` across multiple `Row` creation calls, as this
 /// avoids the allocations involved in `Row::new()`.
-#[derive(Default, Eq, PartialEq, Hash, Serialize, Deserialize)]
+#[derive(Clone, Default, Eq, PartialEq, Hash, Serialize, Deserialize)]
```

Comment:
I'm not sure we call `Row::clone_from` anywhere, but if we do we have to implement our own `Clone` implementation. I'm surprised that we didn't for the current implementation of `Clone`...

Reply:
Updated to manually implement `Clone`, and call through to `CompactBytes` impl of `Clone`, this way we get `clone_from`.